corehq/apps/domain/middleware.py

```diff
@@ -81,3 +82,23 @@ def process_view(self, request, view_func, view_args, view_kwargs):
                     }
                 )
         return None
+
+
+DOMAIN_AUDIT_VIEWS = [
+    'ConfigurableReportView',  # for audit number of configurable report viewed
+    'ProjectReportDispatcher',  # for audit number of nonconfigurable reports viewed
+    'retrieve_download',  # for audit number of downloads
```

Comment:
note that there is another view `soil.retrieve_download` that doesn't have a `domain`: https://github.com/dimagi/commcare-hq/pull/22119

Another reason to use a decorator.